common/operations.go

```diff
@@ -21,47 +22,39 @@ const (
 
 var (
 	Operations = adapter.Operations{
-		InstallBookInfoCommand: &adapter.Operation{
-			Type: int32(meshes.OpCategory_SAMPLE_APPLICATION),
-			Properties: map[string]string{
-				adapter.OperationDescriptionKey:  "Istio Book Info Application",
-				adapter.OperationVersionKey:      "",
-				adapter.OperationTemplateNameKey: "templates/bookinfo.yaml",
-				adapter.OperationServiceNameKey:  "productpage",
-			},
+		BookInfoCommand: &adapter.Operation{
+			Type:        int32(meshes.OpCategory_SAMPLE_APPLICATION),
```

Comment:
We should move away from SAMPLE_APPLICATION and move to WORKLOAD. 

Workloads may be for sample purposes or may be for production or development or testing purposes. The purpose of the workload needs to be a characteristic of a workload.

Reply:
@leecalcote  This one still is working over the existing protobuf. So we are constraint to use existing names.

Reply:
As we write a v2 of the adapters, let's be open to changing that proto (let's change it).